_posts/2017-06-10-dot-net-fringe-portland-oregon-2017.md

```diff
@@ -0,0 +1,77 @@
+image: /images/dot-net-fringe-2017-repeating.png
+---
+
+As a team empowered by .NET open source, we've always been sensitive to the people, projects, and conferences that make the .NET community a joy to be part of. With [.NET Fringe](http://dotnetfringe.org/) being a conference the RIMdev team was looking forward to, we had to go this year. With a mix of backend .NET developers, frontend developers, and quality assurance engineers we made our way to Portland.
+
+## Favorite People Moments
+
+The best part of being a developer in 2017 is access to your community. You can reach out to friends, library authors, and community leaders in ways you couldn't even 5 years ago. What's better than *Twitter Friends*? How about meeting them **in real life?!**.
+
+### Team Bonding
+
+The team I work with is amazing. While we work long eight hour days with each other, it was nice to get a different perspective on individuals. Portland and .NET Fringe brought us closer together.
+
+### Jon Galloway
+
+Jon Galloway does a lot for the .NET Community, and it was an honor to meet him in person. I thanked him for the work he is doing with the [.NET Foundation]() and we also got a chance to take some pictures. Maybe I'll even get a 2018 calendar out of it. [His talk about what the .NET Foundation](https://www.youtube.com/watch?list=PLwZVRWVJepJtK6UZD-m2VLU2k2V-O5OrG&v=W-fqEOqiK78) does is illuminating and I recommend it.
+
+### Tasha, Greg, and Dave
+
+A great moment is when you get a chance to sit and share drinks with individuals you feel you've known for a long time but meeting for the first time. [Tasha](https://twitter.com/TashasEv), [Greg](https://twitter.com/justcallme98), and [Dave](https://twitter.com/daveaglick) are awesome people and strong advocates in the .NET community to stay positive and do your best. Their enthusiasm is infectious. Not to mention, we somehow got talked into wearing bowls on our heads.
+
+### Glenn Block and Troy Howard
+
+The organizers of the conference are passionate individuals and they exude it. It was great talking to them about the growth of the .NET Fringe community and a possible East Coast iteration. If it happens, RIMdev team members will be there to help.
+
+### Elastic People
+
+The elastic team of [Martijn](https://twitter.com/Mpdreamz) and [Michelle](https://twitter.com/miiiiiche) were friendly and knowledgable. They also told me about more training oppurtunities to level up my Elasticsearch skills.
+
+## Favorite Talks
+
+The point of conferences is to absorb as much knowledge from smart individuals. The display of intelligence at .NET Fringe was humbling.
+
+### Distributed Sagas
+
+My favorite talk was given by [Caitie McCaffrey](https://www.youtube.com/watch?v=0UTOLRTwOX0) about distributed sagas and their role in microservices architectures. A sign of any good talk is leaving me with more questions to answer myself through exploration, and this one had our team talking through out the night.
```

Comment:
For consistency you may want to link to a Twitter account here, rather than the talk video.